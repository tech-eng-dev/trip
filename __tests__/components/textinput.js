import React from 'react';
import renderer from 'react-test-renderer';
import CustomInput from '../../src/components/textInput';


test('CustomInput renders correctly', () => {
  const textInput = renderer.create(<CustomInput
    placeholder="test"
    text="test"
    testID="test"
    onChange={() => {}}
    multiline
    width={200}
    keyboardType="default"
    secureTextEntry
    autoCapitalize="WORD"
    border
  />).toJSON();
  expect(textInput).toMatchSnapshot();
});
