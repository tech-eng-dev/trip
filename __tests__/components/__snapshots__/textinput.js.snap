// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomInput renders correctly 1`] = `
<TextInput
  allowFontScaling={true}
  autoCapitalize="sentences"
  autoCorrect={true}
  keyboardType="default"
  maxLength={1024}
  multiline={true}
  onChangeText={[Function]}
  placeholder="test"
  secureTextEntry={true}
  style={
    Array [
      Object {
        "backgroundColor": "#EEEEEE",
        "borderColor": "#5C6063",
        "borderRadius": 4,
        "color": "#5C6063",
        "fontSize": 20,
        "height": 40,
        "marginVertical": 10,
        "paddingHorizontal": 10,
        "paddingVertical": 0,
      },
      Object {
        "borderWidth": 0.5,
        "height": 150,
        "textAlignVertical": "top",
        "width": 200,
      },
    ]
  }
  testID="test"
  underlineColorAndroid="transparent"
  value="test"
/>
`;
